tests/unit/services/maps-test.js

```diff
@@ -0,0 +1,14 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('service:maps', 'Unit | Service | maps', {
+  // Specify the other units that are required for this test.
+  // needs: ['service:foo']
+});
+
+test('should fetch a map element based on location', function(assert) {
```

Comment:
This test is just verifying that the element gets created and properly returned.  My plan is to stub the google util and verify that create is called when not in cache, and skipped when it is.  I also will make sure the pin is called.  This one is actually calling through to google.  not sure we want to do this.